virtualizarr/xarray.py

```diff
@@ -41,9 +41,52 @@ def open_virtual_dataset(
     drop_variables: Iterable[str] | None = None,
     loadable_variables: Iterable[str] | None = None,
     decode_times: bool | None = None,
-    cftime_variables: Iterable[str] | None = None,
     indexes: Mapping[str, xr.Index] | None = None,
 ) -> xr.Dataset:
+    """
+    Open a data file, virtual dataset, Zarr store, etc. as an [xarray.Dataset][] wrapping virtualized zarr arrays.
+
+    No data variables will be loaded unless specified in the ``loadable_variables`` kwarg (in which case they will be xarray lazily indexed arrays).
+
+    Xarray indexes can optionally be created (the default behaviour). To avoid creating any xarray indexes pass ``indexes={}``.
```

Comment:
```suggestion
    Xarray indexes can optionally be created (the default behaviour is to create indexes for any 1D coordinate variables). To avoid creating any xarray indexes pass ``indexes={}``.
```